tests/appsec/test_processor.py

```diff
@@ -17,8 +17,16 @@
 RULES_MISSING_PATH = os.path.join(ROOT_DIR, "nonexistent")
 
 
+def _enable_appsec(tracer):
+    tracer._appsec_enabled = True
+    # Hack: need to pass an argument to configure so that the processors are recreated
+    tracer.configure(api_version="v0.4")
+    return tracer
```

Comment:
needed?

Reply:
yeah, there's no way in the API to enable appsec at the moment

Reply:
sorry, I did not give you enough context

```suggestion
```

is the return needed? you are working from a reference and it doesn't seem like the result is used anywhere? future plans?